src/Microsoft.AspNetCore.Mvc.Razor.Host/IInjectTargetExtension.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Evolution.CodeGeneration;
+
+namespace Microsoft.AspNetCore.Mvc.Razor.Host
+{
+    public interface IInjectTargetExtension : IRuntimeTargetExtension
```

Comment:
Since inject is typically quite an overloaded CS term lets call this `IInjectDirective...`